openbb_platform/core/openbb_core/provider/standard_models/options_chains.py

```diff
@@ -31,30 +31,43 @@ def to_upper(cls, v: str) -> str:
 class OptionsChainsData(Data):
     """Options Chains Data."""
 
-    symbol: Optional[str] = Field(
-        description=DATA_DESCRIPTIONS.get("symbol", "")
-        + " Here, it is the underlying symbol for the option.",
+    underlying_symbol: Optional[str] = Field(
         default=None,
+        description="Underlying symbol for the option.",
+    )
+    underlying_price: Optional[float] = Field(
+        default=None,
+        description="Price of the underlying stock.",
     )
     contract_symbol: str = Field(description="Contract symbol for the option.")
     eod_date: Optional[dateType] = Field(
         default=None, description="Date for which the options chains are returned."
     )
     expiration: dateType = Field(description="Expiration date of the contract.")
+    dte: Optional[int] = Field(
```

Comment:
I'd vote for a more verbose name on the standard model. I'm fine with `dte` but we did want more verbosity.